src/Arcus.Testing.Integration.DataFactory/TemporaryDataFlowDebugSession.cs

```diff
@@ -277,12 +278,17 @@ private async Task StartDataFlowAsync(string dataFlowName, RunDataFlowOptions op
         private DataFlowDebugPackageDebugSettings CreateDebugSettings(RunDataFlowOptions options)
         {
             var settings = new DataFlowDebugPackageDebugSettings();
+
             foreach (KeyValuePair<string, BinaryData> parameter in options.DataFlowParameters)
             {
                 _logger.LogTrace("[Test:Setup] Add DataFlow parameter '{Name}' to debug session", parameter.Key);
                 settings.Parameters[parameter.Key] = parameter.Value;
             }
 
+            _logger.LogTrace("[Test:Setup] Add DataSet parameters to debug session");
```

Comment:
Maybe go over the parameters like we did for the data flow and log for each data set their parameters, otherwise, this logging statement does not add much value.